tests/EventFactoryTest.php

```diff
@@ -92,6 +92,20 @@ public function createWithPayloadDataProvider()
                     ],
                 ],
             ],
+            [
+                [
+                    'message' => 'testMessage @param',
```

Comment:
Assuming that this test case covers the case in which a `message_formatted` param is specified in the payload, please use the `sprintf` notation for the value of the `message` key and a totally different message for the value of the `formatted` key so that we can catch if the override works